docs/troubleshooting/large-partition-table.rst

```diff
@@ -73,24 +75,27 @@ Example output:
 
 .. code-block:: console
 
-   keyspace_name | table_name | sstable_name                                                                       | partition_size | partition_key                                       | compaction_time
-   --------------+------------+------------------------------------------------------------------------------------+----------------+-----------------------------------------------------+--------------------------
-          demodb |       tmcr | /var/lib/scylla/data/demodb/tmcr-cba79c008e4f11e8908a000000000001/la-6-big-Data.db |     1188716932 | {key: pk{000400000001}, token:-4069959284402364209} | 2018-07-23 08:10:34
+   keyspace_name | table_name | sstable_name                                                                       | partition_size | partition_key                                       | rows | compaction_time
+   --------------+------------+------------------------------------------------------------------------------------+----------------+-----------------------------------------------------+------+--------------------------
+          demodb |       tmcr | /var/lib/scylla/data/demodb/tmcr-cba79c008e4f11e8908a000000000001/la-6-big-Data.db |     1188716932 | {key: pk{000400000001}, token:-4069959284402364209} | 1942 | 2018-07-23 08:10:34
 
 
 .. _large-partition-table-configure:
 
 Configure
 ^^^^^^^^^
 
-Configure the detection threshold of large partitions with the ``compaction_large_partition_warning_threshold_mb`` parameter (default: 1000MB) in the scylla.yaml configuration file.
-Partitions bigger than this threshold are reported in the ``system.large_partitions`` table and generate a warning in the Scylla log (refer to :doc:`log </getting-started/logging/>`).
+Configure the detection thresholds of large partitions with the ``compaction_large_partition_warning_threshold_mb`` parameter (default: 1000MB)
+and the ``compaction_rows_count_warning_threshold`` paramater (default 100000)
+in the scylla.yaml configuration file.
+Partitions that are bigger than the size threshold and/or hold more than the rows count threshold are reported in the ``system.large_partitions`` table and generate a warning in the Scylla log (refer to :doc:`log </getting-started/logging/>`).
 
-For example (set to 500MB):
+For example (set to 500MB / 50000, respectively):
 
 .. code-block:: console
 
    compaction_large_partition_warning_threshold_mb: 500
+   compaction_rows_count_warning_threshold: 50000
```

Comment:
Nice, I wasn't aware we added this.
Sad there's duplication and partial information between docs/troubleshooting/large-partition-table.rst which you modify here, and docs/dev/system_keyspace.md. But we don't have to fix that right now.